CMSgov/onemac
# Test coverage

Story: N/A - general test coverage requests
Endpoint: https://d1kyncyto8b3p9.cloudfront.net/

### Details
Cleaned up and expanded some tests, fixed an issue with the attachment links

### Changes
- improved the getAllByAuthorizedTerritories test stub
- added tests to getDetail
- discovered the signed URL functionality in the getDetail had been commented out ? reinstated

### Test Plan

1. test coverage updates: Automated tests run cleanly
2. Attachments update: 
3. Login as a submitter user
4. submit a component with attachments
5. go to package dashboard
6. click to view package details
7. click on attachments
8. verify the attachment is downloaded/displayed.

NOTE: old data may or may not have valid attachments, seed data does NOT have valid attachments
